dispatch/static/manager/src/js/components/PollEditor/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import pollsActions from '../../actions/PollsActions'
+import PollForm from './PollForm'
+
+import ItemEditor from '../ItemEditor'
+
+const TYPE = 'Poll'
+const AFTER_DELETE = 'polls'
+
+const mapStateToProps = (state) => {
+  return {
+    listItem: state.app.polls.single,
+    entities: {
+      remote: state.app.entities.polls,
+      local: state.app.entities.local.polls,
+    },
+    token: state.app.auth.token
+  }
+}
+
+function prepareJSONData(data) {
+  data.answers_json = data.answers
+  return data
+}
+
+const mapDispatchToProps = (dispatch) => {
```

Comment:
To be consistent with the rest of the codebase, put `mapStateToProps` and `mapDispatchToProps` after the definition for `PollEditorComponent`